scripts/restructure-ideas-image-urls.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../src/db');
+
+async function updateImageUrls() {
+
+    try {
+
+        const ideas = await db.Idea.findAll();
+
+        console.log(`Going through all ideas to look for image-arrays including the URLs with the base ${process.env.IMAGE_URL_OLD_BASE_PATH}, to replace them with the URL base ${process.env.IMAGE_URL_NEW_BASE_PATH}`)
+
+        for (let idea of ideas) {
+            let updatedImageUrls = idea.extraData.images
+                .map(url => url.replace(process.env.IMAGE_URL_OLD_BASE_PATH, process.env.IMAGE_URL_NEW_BASE_PATH))
+            
+            if (updatedImageUrls.length === 0) {
```

Comment:
You can actually test this before doing the loop. Because the mapping now touches all the urls, we can check simple check idea.extraData.images.length.

Reply:
I don't think so, because in theory there might be images with another URL, that are not updated. So, in that case, there are URL's in the array, but there is no need to save them to the database

Reply:
True! Changing it